app/workers/html_variant_success_create_worker.rb

```diff
@@ -0,0 +1,9 @@
+class HtmlVariantSuccessCreateWorker
+  include Sidekiq::Worker
+
+  sidekiq_options queue: :default, retry: 10
+
+  def perform(html_variant_id, article_id)
```

Comment:
I'm sticking to the loose theme of if it's 2 or fewer arguments and it's identifiable what the arguments are when this worker is called, we're not going to use a Hash.

I definitely think we gain a bit by going back through all the workers once they're moved over with a more strict guideline - happy to do that!

I'm also happy to change this to a Hash now if people have a strong preference for that.